tuneme/urls.py

```diff
@@ -21,6 +21,10 @@
 
     url(r'', include('molo.core.urls')),
     url(r'', include(wagtail_urls)),
+
+    url(r'', include('molo.profiles.urls',
+                namespace='molo.profiles'))
+                
```

Comment:
Also there's a `pep8` violation here, the `namespace` needs to line up with the opening bracket of `include(`.